src/resampling.jl

```diff
@@ -129,3 +130,221 @@ Upsamples by a factor of two and applies the abs2 operation. The code is optimiz
 function upsample2_abs2(mat::AbstractArray{T, N}; dims=1:N) where {T,N}
     return abs2.(upsample2(mat, dims=dims))
 end
+
+"""
+    resample_czt(arr, rel_zoom; shear=nothing, shear_dim=nothing, fix_nyquist=false, new_size = size(arr), rel_pad=0.2)
+
+resamples the image with fixed factors or a list of separable functions using the chirp z transform algorithm.
+The data is first padded by a relative amount `rel_pad` which is needed to avoid wrap-around problems.
+As opposed to `resample()`, this routine allows for arbitrary non-integer zoom factors.
+It is reasonably fast but only allows a stretch (via `rel_zoom`) and a shift (via `shear` in pixels) per line or column
+
+normalize=true by default multiplies by an appropriate factor so that the array size is included in the scaling. 
```

Comment:
`normalize=true`

Reply:
I mean also the formatting in line 152. Please code related text in ` `

Reply:
`

Reply:
I removed this comment as the function does not have a normalize argument.